.github/workflows/ci-automated-pipeline.yaml

```diff
@@ -0,0 +1,155 @@
+jobs:
+  auto-pr-triage:
+    runs-on: ubuntu-latest
+    name: Triage for automated PRs
+    if: |
+      contains(github.event.pull_request.title, 'Release') &&
+      github.actor == 'bitnami-bot'
+    steps:
+      # Enables auto-merge and adds necessary labels for automated releases' PRs
+      - id: labeling
+        name: Label PR
+        uses: andymckay/labeler@1.0.4
+        with:
+          repo-token: "${{ secrets.GITHUB_TOKEN }}"
+          add-labels: "verify, auto-merge"
+      - id: auto-merge
+        name: Enable auto-merge
+        run: |
+          curl --request POST \
+          --url https://api.github.com/graphql \
+          --header 'authorization: Bearer ${{ secrets.BITNAMI_BOT_TOKEN }}' \
+          --data '{
+            "query": "mutation { enablePullRequestAutoMerge(input: {pullRequestId: \"${{ github.event.pull_request.node_id }}\", mergeMethod: SQUASH}) { clientMutationId }}"
+            }' \
+          --fail
+  get-container:
+    runs-on: ubuntu-latest
+    name: Get modified containers
+    needs: auto-pr-triage
+    outputs:
+      container: ${{ steps.get-container.outputs.container }}
+      result: ${{ steps.get-container.outputs.result }}
+      path: ${{ steps.get-container.outputs.path }}
+    steps:
+      - id: get-container
+        name: Get modified containers
+        run: |
+          # Using the Github API to detect the files changed as git merge-base stops working when the branch is behind
+          # and jitterbit/get-changed-files does not support pull_request_target
+          URL="https://api.github.com/repos/${{ github.repository }}/pulls/${{ github.event.pull_request.number }}/files"
+          files_changed_data=$(curl -s --header 'authorization: Bearer ${{ secrets.GITHUB_TOKEN }}' -X GET -G "$URL")
+          files_changed="$(echo $files_changed_data | jq -r '.[] | .filename')"
+          # Adding || true to avoid "Process exited with code 1" errors
+          containers_dirs_changed="$(echo "$files_changed" | xargs dirname | grep -o "^bitnami/[^/]*" | sort | uniq || true)"
+          flavors=($(echo "$files_changed" | xargs dirname | grep -o "^bitnami/[^/]*/[^/]*/[^/]*" | sort | uniq || true))
+          container_name=$(echo "$containers_dirs_changed" | sed "s|bitnami/||g")
+
+          if [[ "${#flavors[@]}" -eq "1" ]]; then
+            # Changes done in only one container -> OK
+            echo "::set-output name=result::ok"
+            echo "::set-output name=container::${container_name}"
+            echo "::set-output name=path::${flavors[0]}"
+          else
+            # No changes detected or there are changes done in more than one container -> FAIL. It doesn't make sense in automated PRs.
+            echo -e "::set-output name=error::No changes detected or there are changes done in more than one container:\n${containers_dirs_changed}\n. The rest of the tests will be skipped."
+            echo "::set-output name=result::fail"
```

Comment:
This is a minor thing but is there a need to use the `output.result` here? Given that `get-container` can only succeed or fail, I think we can just make the job finish (when it succeeds) or fail (maybe a `exit 1`). That way we avoid both the output param and the `show-error` job.

Reply:
You right!